repos/system_upgrade/el7toel8/actors/systemfacts/libraries/systemfacts.py

```diff
@@ -269,3 +269,7 @@ def _get_firewall_status(service_name):
         iptables=_get_firewall_status('iptables'),
         ip6tables=_get_firewall_status('ip6tables'),
     )
+
```

Comment:
lint is complaining that needs another blank line here

Reply:
fixed